src/components/toast-notifications.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Notification, { notify } from 'react-notify-toast'
+
+import { useAuth } from '@/auth/use-auth'
+import { theme } from '@/theme'
+
+export function ToastNotifications() {
```

Comment:
This should probably use `ToastContext` in the future, right?

Reply:
it used to already, but I opted to use the simplest implementation for now and add in context etc. back in later when I understand the requirements of other notifications.

Reply:
Currently, `ToastContext` is not in use and you only handle auth notifications.

Reply:
yes.

Reply:
removed unused file.